config/generateConfigFile.cmake

```diff
@@ -22,12 +22,10 @@ set(EXV_UNICODE_PATH     ${EXIV2_ENABLE_WIN_UNICODE})
 check_function_exists( gmtime_r EXV_HAVE_GMTIME_R )
 check_function_exists( mmap     EXV_HAVE_MMAP )
 check_function_exists( munmap   EXV_HAVE_MUNMAP )
-check_function_exists( strerror     EXV_HAVE_STRERROR )
+check_function_exists( strchr   EXV_HAVE_STRCHR )
```

Comment:
Why are you checking whether `strchr` exists? That is required by C89 & C99.

(I am just puzzled, since it's not used in this file.)

Reply:
Oh, damn it. I had a conflict when I rebased in top of master after I merged #294 , and I did not solve it correctly. Good catch! I will fix it and push the changes again.

Reply:
Fixed. I will merge once the CI do its job ;)